pages/_document.tsx

```diff
@@ -15,7 +16,7 @@ export default class MyDocument extends Document {
 					<meta charSet="utf-8" />
 					<meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no" />
 					{/* PWA primary color */}
-					<meta name="theme-color" content={'placeholder'} />
+					<meta name="theme-color" content={blue[500]} />
```

Comment:
I think the simplist solution here is to move meta theme-color from _document to _app. Is there any reason why this would be a bad idea?

Reply:
It seems like it doesn't make sense with the way the TSX files are structured. _document contains the meta tags, _app shouldn't contain any. At any rate, blue[500] is valid since it's the default theme value anyway, and crawlers won't have a custom theme active.

Reply:
_app gets evaluated on build too, so there's no reason not to include <meta> tags there if they _have_ to go there. <title> is already set there for example.